packages/cli/src/models/status/StatusReport.ts

```diff
@@ -0,0 +1,20 @@
+import { Logger } from 'zos-lib';
+
+export default class StatusReport {
+
+  public expected: string;
```

Comment:
We should start taking advantage of encapsulation. I don't see a reason to use getters and setters if not necessary (i.e., if an attribute is accessed from the outside, I think it's ok to define it as public) but if it's not accessed from the outside, let's define them as `private` or `protected` as appropiate.

Reply:
Yes agreed. I am considering encapsulation something to be addressed formally after the first phase of the migration. For now, I'm making everything public unless I see an underscore at the beginning of the member name. I believe that that is a convention in the code. 
Why do I want to do it later? Once the whole codebase is ported to ts, tools can very easily find the usages of a function in Typescript, while it's quite difficult in Javascript.